fdb-record-layer-core/src/test/proto/test_records_text.proto

```diff
@@ -34,7 +34,7 @@ message SimpleDocument {
 
 message ComplexDocument {
     optional int64 group = 1;
-    optional int64 doc_id = 2;
+    optional int64 doc_id = 2 [(field).primary_key = true];
```

Comment:
The primary key for a `ComplexDocument` should be `concat("group", "doc_id")`. I think this should mention that the PK is set in the test, unless there is one that's missing it.